doc/telescope.txt

```diff
@@ -2367,7 +2339,7 @@ work and make it really easy create new previewers.
 
 Furthermore, there are a collection of previewers already defined which can be
 used for every picker, as long as the entries of the picker provide the
-necessary fields. The more important ones are
+necessary fields. The more important once are
```

Comment:
I don't think this was a typo

Reply:
This is not a change I've made, but I guess must have got intertwined with the state of `master` this branch is based off of.

Thanks for pointing it out! I'll merge sure this is untouched when I rebase prior to merging this PR (hopefully soon :)